pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1158,14 +1158,25 @@ public void PauseForCoverShot()
             Thread.Sleep(1000); // Give windows time to repaint
             var coverSavePath = GetCoverShotPath();
             ScreenshotManager.TakeNextShot(SkylineWindow, coverSavePath, ProcessCoverShot);
+            string coverSavePath2 = null;
```

Comment:
I am going to need to understand what you are doing here and why. I have used what existed before to create all necessary covershots, and I am not sure this represents an improvement. It looks like you can't get through without pausing the test, which seems like a bad idea, because it would require me to sit around and tend the machine while it is taking 60+ covershots. (20 x English, Japanese, Chinese)

Reply:
Perhaps this is a misnomer then:
![image](https://user-images.githubusercontent.com/1200761/97643858-058a2e80-1a06-11eb-8e68-d41078466190.png)
Let's make that "Take cover shots" and not have it pause.

Reply:
When I think about the ultimate UI for this, which it seems like we are not engaged in designing, I think it would be nice to be able to have the option to either pause at the moment of taking the covershot, which makes it easier to tweak, e.g. adjust the layout, File > Save, and grab the .view file, or just take screenshots in bulk. I guess I personally transitioned from producing covershots to taking them all in bulk, and kinda rushed to get back to other work once this largish undertaking was complete. So, as you are pointing out, I didn't leave it in an ideal condition for the next single covershot to be added.

Reply:
So another radio button "Update cover shots"?

Reply:
I guess the way it was intended to function was that it would only pause if you had not set the CoverShotName, which gets set in the test constructor. So, if you wanted to pause the test, you would comment out the CoverShotName, much the way you would set IsPauseForScreenShots = true to pause for screenshots when not using SkylineTester. I think this is still a fine implementation, provided it can be better commented and understood by new developers.

Reply:
You can rename it and even move it if you want. Maybe "Covershot mode" after "Demo mode"

Reply:
I don't think we need UI for deciding whether to pause or take a snapshot. I started thinking it would be pausing, hence most of the "pause" use and then decided that would be way too much work to do for all 20 x 3 languages x 2 formats. So, I guess the main intent for how you leave your test when you commit is fully automated without any pause. When you are developing it, you likely want to comment out its name and just have it pause in order to be able to tweak the layout, as I expect to do at some point for your test. Since, that feels more like a development time thing, I think it is fine to require the developer to comment out the name.

Reply:
If you care enough, you can change all the language including the code to be clearer. e.g. "IsCoverShotMode" etc. Where covershot mode should pause if there is no CoverShotName and otherwise automatically take and store the screenshot at files based on the name.